docs/disclaimer.md

```diff
@@ -0,0 +1,30 @@
+# Disclaimer Service
+
+The Disclaimer Service is a component at the service layer of the BDK which aims to cover the Connections part of the [REST API documentation](https://developers.symphony.com/restapi/reference).
```

Comment:
_which aims to cover the **Connections** part_,   looks like it should be **Disclaimers**

Reply:
@symphony-soufiane did you change something regarding my comment?